anchor/programs/sponsor-relayer/src/instructions/authorize_relayer.rs

```diff
@@ -0,0 +1,19 @@
+use anchor_lang::prelude::*;
+
+use crate::{Relayer, Sponsorship};
+
+#[derive(Accounts)]
+pub struct AuthorizeRelayer<'info> {
+    #[account()]
+    pub authority: Signer<'info>,
```

Comment:
just so I understand: the authority is the authority who can update this authorization?

Reply:
authority here is the same one who created the sponsorship account (using the initialize_sponsorship ix) and is going to authorize this relayer (if the initialize_sponsorship set relayer to permissioned)